### PR TITLE
Update wording around date equality test

### PR DESCRIPTION
**Pull request by GillesDebunne**
As discussed in #108

### CODE REVIEW

File: docs/math.md [hunk 1]
```diff
@@ -98,10 +98,11 @@ DateTime implements `#valueOf` to return the epoch timestamp, so you can compare
 d1 < d2 // is d1 before d2?
 ```
 
-However, `===` compares object identity (not a useful concept in a library with immutable types) and `#equals` compares both the time and additional metadata, such as the locale and time zone. If you're only interested in checking the equality of the timestamps, you need to coerce them:
+However, beware that `===` compares object identity, which not a useful concept in a library with immutable types. Use `#equals` to compare both the time and additional metadata, such as the locale and time zone. If you're only interested in checking the equality of the timestamps, you can use:
 
 ```js
-+d1 === +d2 // are d1 and d2 the same instant in time?
+d1.toMillis() === d2.toMillis() // are d1 and d2 the same instant in time?
++d1 === +d2 // same test, using object coercision
```
Comment (icambron) [inline docs/math.md:105]: coercion

File: docs/math.md [hunk 2]
```diff
@@ -98,10 +98,11 @@ DateTime implements `#valueOf` to return the epoch timestamp, so you can compare
 d1 < d2 // is d1 before d2?
 ```
 
-However, `===` compares object identity (not a useful concept in a library with immutable types) and `#equals` compares both the time and additional metadata, such as the locale and time zone. If you're only interested in checking the equality of the timestamps, you need to coerce them:
+However, beware that `===` compares object identity, which not a useful concept in a library with immutable types. Use `#equals` to compare both the time and additional metadata, such as the locale and time zone. If you're only interested in checking the equality of the timestamps, you can use:
```
Comment (icambron) [inline docs/math.md:101]: which is not

Comment (icambron) [inline docs/math.md:101]: Also you want "be aware that"; "beware that" doesn't sound quite right

Comment (GillesDebunne) [inline docs/math.md:101]:
```suggestion
However, be aware that `===` compares object identity, which is not a useful concept in a library with immutable types. Use `#equals` to compare both the time and additional metadata, such as the locale and time zone. If you're only interested in checking the equality of the timestamps, you can use:
```

File: docs/math.md (hunk 1 above)
Comment (GillesDebunne) [inline docs/math.md:105]:
```suggestion
+d1 === +d2 // same test, using object coercion
```

**Pull request merged**